src/main/proto/service.proto

```diff
@@ -1,5 +1,7 @@
 syntax = "proto3";
 
+package tech.sourced.featurext;
```

Comment:
Clearly grouping generated code together is convenient for both, humans and machines i.e for excluding from linters/formatters, so why do not we put it in `tech.sourced.featurext.generated`, what do you think?

Reply:
yep. The original idea was featurext won't have any non-generated code. But it might. So changed. 👍